processor/src/it/java/com/linecorp/decaton/processor/PerformanceBoundTest.java

```diff
@@ -1,93 +0,0 @@
-/*
- * Copyright 2020 LINE Corporation
- *
- * LINE Corporation licenses this file to you under the Apache License,
- * version 2.0 (the "License"); you may not use this file except in compliance
- * with the License. You may obtain a copy of the License at:
- *
- *   https://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
- * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
- * License for the specific language governing permissions and limitations
- * under the License.
- */
-
-package com.linecorp.decaton.processor;
-
-import static org.hamcrest.Matchers.greaterThanOrEqualTo;
-import static org.junit.Assert.assertThat;
-
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-
-import org.junit.Assume;
-import org.junit.Before;
-import org.junit.Test;
-
-import com.linecorp.decaton.benchmark.Benchmark;
-import com.linecorp.decaton.benchmark.BenchmarkConfig;
-import com.linecorp.decaton.benchmark.BenchmarkResult;
-import com.linecorp.decaton.benchmark.DecatonRunner;
-
-public class PerformanceBoundTest {
```

Comment:
just `@Ignore` it?

Reply:
That also works, but since I'm not sure if we'll keep this way of testing performance (run as part of integration test), I wanted to clean things up once and add it back later if necessary to avoid leaving dead code.